fundamentals/fundamentals-collections/src/main/scala/izumi/functional/IzEitherAggregations.scala

```diff
@@ -0,0 +1,101 @@
+package izumi.functional
+
+import izumi.functional.IzEitherAggregations.*
+import izumi.fundamentals.collections.nonempty.NonEmptyList
+
+import scala.collection.compat.*
+import scala.language.implicitConversions
+
+trait IzEitherAggregations {
+  @inline implicit final def EitherBiAggregate[L, R, Col[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], R]]
+  ): EitherBiAggregate[L, R, Col] = new EitherBiAggregate(col)
+  @inline implicit final def EitherBiFlatAggregate[L, R, Col[x] <: IterableOnce[x], Col2[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], Col2[R]]]
+  ): EitherBiFlatAggregate[L, R, Col, Col2] = new EitherBiFlatAggregate(col)
+  @inline implicit final def EitherBiFind[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFind[Col, T] = new EitherBiFind(col)
+  @inline implicit final def EitherBiFoldLeft[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFoldLeft[Col, T] = new EitherBiFoldLeft(col)
+  @inline implicit final def EitherBiMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiMapAggregate[Col, T] = new EitherBiMapAggregate(col)
+  @inline implicit final def EitherBiFlatMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFlatMapAggregate[Col, T] = new EitherBiFlatMapAggregate(col)
+}
+
+object IzEitherAggregations extends IzEitherAggregations {
+
+  final class EitherBiAggregate[L, R, Col[x] <: IterableOnce[x]](private val col: Col[Either[NonEmptyList[L], R]]) extends AnyVal {
+    /** `sequence` with error accumulation */
+    def biAggregate(implicit b: Factory[R, Col[R]]): Either[NonEmptyList[L], Col[R]] = {
+      new IzEither.EitherBiAggregate(col)
+        .biAggregate(b, _.toList)
+        .left.map(NonEmptyList.unsafeFrom)
+    }
+
+    /** `sequence_` with error accumulation */
+    def biAggregateVoid: Either[NonEmptyList[L], Unit] = {
+      new IzEither.EitherBiAggregate(col)
+        .biAggregateVoid(_.toList)
+        .left.map(NonEmptyList.unsafeFrom)
+    }
+  }
+
+  final class EitherBiFlatAggregate[L, R, Col[x] <: IterableOnce[x], Col2[x] <: IterableOnce[x]](private val result: Col[Either[NonEmptyList[L], Col2[R]]]) extends AnyVal {
+    /** `flatSequence` with error accumulation */
+    def biFlatAggregate(implicit b: Factory[R, Col[R]]): Either[NonEmptyList[L], Col[R]] = {
+      new IzEither.EitherBiFlatAggregate(result)
+        .biFlatAggregate(b, _.toList)
+        .left.map(NonEmptyList.unsafeFrom)
+    }
+  }
+
+  final class EitherBiFind[Col[x] <: IterableOnce[x], T](private val s: Col[T]) extends AnyVal {
+    def biFind[E](predicate: T => Either[NonEmptyList[E], Boolean]): Either[NonEmptyList[E], Option[T]] = {
```

Comment:
IzEither.biFind does not use the fact that left side is a List, so its signature can just be changed to:
```scala
    def biFind[E](predicate: T => Either[E, Boolean]): Either[E, Option[T]] = {
```
However, that means there's no need for a NonEmptyList version of biFind, you may keep a conversion here for convenience though
```suggestion
  @inline implicit final def EitherBiFind[Col[x] <: IterableOnce[x], T](col: Col[T]): IzEither.EitherBiFind[Col, T] = new IzEither.EitherBiFind(col)
```